HW1 Crystal Resume Webpage.html

```diff
@@ -0,0 +1,130 @@
+<!DOCTYPE html>
+<html>
+<html lang="en">
+<head>
+<style>
+h1 {
+    font-family: "Calibri";
+	text-align: center;
+}
+
+p {
+    font-family: "Calibri";
+	background-color:#999999;
+	color: #FFFFFF;
+	text-align: center;
+
+}
+body {
+    background-image: url("background.jpg");
+	background-size: contain;
+	background-repeat:no-repeat;
+	width:1024px;
+	height:2000px;
+}
+div.picture1 {
+   width:135px;
+   height:158px;
+   background-image:url("profile.jpg");
+   background-size: contain;
+   background-repeat:no-repeat;
+   background-color: white;
+   border: solid white 4px;
+   float: left;
+
+}
+.container {
+      background-color: #add8e6;
+	  border-color: #999999;
+	  border-style: solid;
+	  border-width: medium;
+      width:30%;
+	  color: #2f4f4f;
+}
+ 
+ .navbar {
+      margin-bottom: 0;
+      border-radius: 0;
+}
+
+footer {
+      background-color: #f2f2f2;
+      padding: 25px;
+    }
+p.fl {
+    font-size: 150%;
+}
+
+
+</style>
+
+<!-- Latest compiled and minified CSS -->
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+
+<!-- jQuery library -->
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+
+
+<title> Greetings!
+</title>
+
+</head>
+
+<body>
+
+
+<p>
+2181 Elderwood Rd<br>Houston, TX<br>
+832-252-1337 <br>crystal.lee@gmail.com
+
+<div class="container text-center">    
+  <h3>Crystal Lee</h3>
+  <br>
+  <div class="row">
+    <div class="col-sm-2">
+      <img src="profile.jpg" class="img-responsive" style="width:100%" alt="Image">
+    </div>
+    <div class="col-sm-10">
+      <div class="well">
+       <p class="fl">Motivated scientist looking for project management role</p>
+      </div>
+    </div>
+  </div>
+</div>
+
+
+<p class="container"; align="left">
```

Comment:
Delete the `;` in between properties. Also delete `align="left"` because it's not a valid property.